tests/test_fx/test_tracer/test_timm_model/test_timm_model.py

```diff
@@ -42,6 +43,7 @@ def trace_and_compare(model_cls, data, output_transform_fn, meta_args=None):
             f'{model.__class__.__name__} has inconsistent outputs, {fx_output_val} vs {non_fx_output_val}'
 
 
+@pytest.mark.skipif(torch.__version__ < '1.12.0', reason='torch version < 12')
```

Comment:
You cannot compare like this, you should use `packaging.version` to compare instead.

Reply:
okay